release-notes/release-notes57.md

```diff
@@ -37,7 +37,12 @@ The following new features were added for the {{site.productname}} 5.7 release.
 
 The following enhancements were made for the {{site.productname}} 5.7 release.
 
-### Enhancement name
+### Improved behavior when using `lists` plugin on mixed content
+
+The `lists` plugin has been updated to improve the handling of content containing:
```

Comment:
Should this say something along the lines of "when converting content to lists"? As atm it's not very clear what "handling" is referring to here.